internal/command/flags.go

```diff
@@ -31,26 +30,19 @@ var (
 )
 
 func addFlagAPIRoot(fs *flag.FlagSet) {
-	fs.StringVar(&flagAPIRoot, "api-root", "", "location of googleapis repository")
+	fs.StringVar(&flagAPIRoot, "api-root", "", "(Required) location of googleapis repository")
```

Comment:
I don't _think_ this is required (see @jskeet's comments on https://github.com/googleapis/generator/pull/25/files)